decidim-admin/app/views/decidim/admin/participatory_processes/show.html.erb

```diff
@@ -30,11 +30,22 @@
     :slug,
     :short_description,
     :description
-    %>
+  %>
+
   <dt><%= display_label(@participatory_process, :hero_image) %></dt>
-  <dd><%= image_tag @participatory_process.hero_image.url.to_s %></dd>
+  <dd>
+    <% if @participatory_process.hero_image %>
+      <%= image_tag @participatory_process.hero_image.url.to_s %>
```

Comment:
Even if you don't set a default image, `hero_image` returns an object that responds to URL. If the image exists, `url` returns something, otherwise returns `nil`. This is what is currently printing (before your changes):

![](https://i.imgur.com/GUKDb5D.png)

So just checking for `process.hero_image` will not change anything, it will still show what I posted here. What about checking if `hero_image.present?` or similar? I'm sure Carrierwave has a way to know if the object is empty or not.